src/iop/rgbcurve.c

```diff
@@ -1457,7 +1457,7 @@ void gui_cleanup(struct dt_iop_module_t *self)
 void init_pipe(struct dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_pixelpipe_iop_t *piece)
 {
   // create part of the pixelpipe
-  dt_iop_rgbcurve_data_t *d = (dt_iop_rgbcurve_data_t *)malloc(sizeof(dt_iop_rgbcurve_data_t));
+  dt_iop_rgbcurve_data_t *d = (dt_iop_rgbcurve_data_t *)dt_alloc_align(64, sizeof(dt_iop_rgbcurve_data_t));
```

Comment:
I don't think this is needed / helpful in this context. Only possible some fields in the struct have to be properly aligned. Here we only ensure that the first field (table) is 64 bit aligned.

Reply:
Sure. From reading the code i suspected that only the table was required to be aligned.